examples/junit/src/test/java/com/example/FuzzTestWithCrashTest.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2025 Code Intelligence GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example;
+
+import com.code_intelligence.jazzer.api.FuzzedDataProvider;
+import com.code_intelligence.jazzer.api.FuzzerSecurityIssueMedium;
+import com.code_intelligence.jazzer.junit.FuzzTest;
+import java.util.Objects;
+
+public class FuzzTestWithCrashTest {
+  @FuzzTest(maxDuration = "10s")
+  void crashFuzz(FuzzedDataProvider data) {
+    String input = data.consumeRemainingAsString();
+    if (Objects.equals(input, "crash")) {
+      throw new FuzzerSecurityIssueMedium();
```

Comment:
This test here might not reproduce the originally reported bug.

It is necessary that the fuzz test calls `Jazzer#reportFindingFromHook`[^1] (respectively uses a sanitizer / hook which does this), because only then `JazzerInternal#lastFinding` is set. Directly throwing an exception here will not cause that.

At least that is my understanding of it.

[^1]: Based on the documentation is sounds as if fuzz tests should not directly call `reportFindingFromHook`. In that case maybe one of the simple sanitizer test cases (e.g. for Regex or Unsafe) could be copied here?